client/src/test/java/io/pravega/keycloak/client/KeycloakAuthzClientTest.java

```diff
@@ -182,16 +193,27 @@ public void builderSetAudience() {
         assertEquals("builder_setAudience", supplier.configuration.getResource());
     }
 
+    @Test
+    public void builderSetAudienceFromString() {
+        TestSupplier supplier = new TestSupplier();
+        KeycloakAuthzClient.builder().withAuthzClientSupplier(supplier).withConfigString(SVC_ACCOUNT_JSON_STRING)
+                .withAudience("builder_setAudience").build();
+        assertEquals("builder_setAudience", supplier.configuration.getResource());
+    }
+
     @Test(expected = KeycloakConfigurationException.class)
     public void builderNoConfig() {
         TestSupplier supplier = new TestSupplier();
         KeycloakAuthzClient.builder().withAuthzClientSupplier(supplier).build();
     }
 
     @Test
-    public void builderAuthenticator() {
+    void builderAuthenticator(boolean isFile) {
         TestSupplier supplier = new TestSupplier();
-        KeycloakAuthzClient.builder().withAuthzClientSupplier(supplier).withConfigFile(SVC_ACCOUNT_JSON_FILE).build();
+        if (isFile)
```

Comment:
Just a nit to have brackets for the if and else block even with just one instruction.